src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/CodeStyle/CSharpIdeCodeStyleOptions.cs

```diff
@@ -72,6 +72,7 @@ internal sealed class CSharpIdeCodeStyleOptions : IdeCodeStyleOptions, IEquatabl
 
     // the following are also used in code generation features, consider sharing:
```

Comment:
Is this comment inaccurate with the addition of the new option? I don't see what code generation features are using the option.